Formula/libb64.rb

```diff
@@ -0,0 +1,20 @@
+class Libb64 < Formula
+  desc "Base64 encoding/decoding library"
+  homepage "https://libb64.sourceforge.io/"
+  url "https://downloads.sourceforge.net/project/libb64/libb64/libb64/libb64-1.2.src.zip"
+  sha256 "343d8d61c5cbe3d3407394f16a5390c06f8ff907bd8d614c16546310b689bfd3"
+
+  def install
+    system "make"
+    bin.mkpath
+    bin.install "base64/base64"
```

Comment:
macOS provides a `base64` command that this will shadow. This formula should be `keg_only` or should only install the `base64` binary into libexec.

```suggestion
    libexec.install "base64/base64"
```

I'm not sure which one is appropriate. It looks like base64 encoding is provided within macOS frameworks:

```sh
otool -L /usr/bin/base64
/usr/bin/base64:
        /System/Library/PrivateFrameworks/CoreDaemon.framework/Versions/B/CoreDaemon (compatibility version 3.0.0, current version 3.0.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.250.1)
```

Should this be keg only if accepted?

Reply:
> Should this be keg only if accepted?

Yes.